assets/css/smallscreen.css

```diff
@@ -0,0 +1,92 @@
+@media screen and (max-width: 600px) {
+  body {
+    margin: 0;
+    background-color: rgb(255, 255, 255);
+  }
+
+  html {
+    font-family: "Lucida Sans", sans-serif;
+  }
+
+  .header {
+    padding: 17px 15px 15px 15px;
+    height: 72px;
+    background-color: #3c3a39;
+    color: #acb7c3;
+    position: relative;
+    max-width: 100vw;
+  }
+
+  .header nav {
+    top: 10px;
+    flex-direction: row;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: nowrap;
+    align-items: center;
+  }
+
+  .header nav .logo {
+    font-size: 20px;
+  }
+
+  .header nav .menu-icon {
+    height: 30px;
+    font-size: 40px;
+  }
+
+  .hero {
+    border: 0;
+    max-width: 100vw;
+    margin: 0;
+    padding-top: 20px;
+    padding-left: 20px;
+    padding-right: 20px;
+    background-color: #1c1a19;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: bottom;
+    background-image: url('https://lh5.googleusercontent.com/proxy/w3oh8vPkBYObX9gqYJpOXMA-lraeW32SsT_6fLv032xO3OwQBeKWRWzovY9c-CLwONIsT4R3VTeZAWeDUg0f2NAYh0SEWONVpJWf-mtClY6BcbPIZKd2TEQ=s0-d');
```

Comment:
- [ ] Please remove this background image from the hero section and replace it with the vector images on the Figma templates and make sure your design matches the template.

Your design

![Screenshot (57)](https://user-images.githubusercontent.com/83803414/153379424-16b66520-63fe-4bcd-8264-485280d42d15.png)

Figma design

![Screenshot (58)](https://user-images.githubusercontent.com/83803414/153379465-2f12ad51-4053-4a3d-9691-975421e64302.png)

Reply:
Thanks for your comment